packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -45,17 +49,20 @@ export interface TabsProps extends Omit<AriaTabsProps, 'className' | 'style' | '
    * The amount of space between the tabs.
    * @default 'regular'
    */
-  density?: 'compact' | 'regular'
+  density?: 'compact' | 'regular',
+  /**
+   * If the tabs should only display icons and no text.
+   */
+  iconOnly?: boolean
```

Comment:
Is there a way to avoid this?

Reply:
Not really, we need people to provide both Icon and Text or Text only in order to have something meaningful for the Picker to display as options. Otherwise we have to try to use the aria-label or some other proxy which may or may not have good formatting or make sense to a sighted user.